adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/ReleaseVersionResolver.kt

```diff
@@ -36,6 +61,9 @@ object ReleaseVersionResolver {
             .sorted()
             .toList()
             .toTypedArray()
+
+        val obsoleteReleases: Array<Int> = getObsoleteReleases() ?: arrayOf(9, 10)
```

Comment:
wondering if this should actually fail if no obsolete releases are found? if this info is depended on and stops working meaning we default back to version 9 and 10 that could cause some unwanted side effects for software/processes that depend on the full list of obsolete versions - for me, if we don't find any obsolete releases it indicates a failure state and we probably want t know about it, not fallback quietly to a different set of versions.

Reply:
Done. Although it used my 2nd account as I forgot that's what my current computer is set to 😅